.github/workflows/metrics.yaml

```diff
@@ -0,0 +1,21 @@
+name: Scheduled Metrics
+on:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  - cron:  '0 0 * * *'
+
+jobs:
+  metrics:
+    name: Collect and Send Metrics
+    runs-on: ubuntu-20.04
+    env:
+      SEGMENT_WRITE_KEY: ${{ secrets.SEGMENT_WRITE_KEY }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python 3.x Part 1
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+      - name: Release Metrics
+        run: python3 scripts/src/metrics/metrics.py
```

Comment:
What if this step fails due to issue? I mean do we have to monitor the cron job status daily. I think it will be good if we can send a slack notification in our slack channel in case of a failure. WDYT? Similar to nightly test here : https://github.com/openshift-helm-charts/development/pull/94/files#diff-a834de30406c9a60f1dfe83be55fb5c928bbc6a8572921d56aa834a8b5e29f24R37

Reply:
That could be a good addition. As far as Segment and Woopra go, they won't be affected by failed cron jobs.

Reply:
Yes I understand that segment and woopra won't be affected. But how do we know, if due to some change this workflow broke and scheduled job stopped working. Slack notification would help in this kind of scenario I think.